pkg/requester/endpoint.go

```diff
@@ -121,6 +121,13 @@ func (e *BaseEndpoint) SubmitJob(ctx context.Context, data model.JobCreatePayloa
 		CreateTime:  job.Metadata.CreatedAt.UnixNano(),
 		ModifyTime:  job.Metadata.CreatedAt.UnixNano(),
 	}
+
+	err = e.store.CreateEvaluation(ctx, *eval)
+	if err != nil {
+		log.Ctx(ctx).Error().Err(err).Msgf("failed to save evaluation for job %s", jobID)
+		return job, err
+	}
+
 	err = e.evaluationBroker.Enqueue(eval)
```

Comment:
since the requester is creating an evaluation in the datastore, it doesn't need to enqueue the evaluation to the broker since there should be a watcher that polls from the data store and enqueues to the broker. This goes for all other evaluations here

Though it would be nice to create the job and evaluation in the datastore in a single transaction. I understand that is not easy to do now, and we can delay it to a later stage. A TODO would be nice for now :)